src/components/images/CAvatar.story.vue

```diff
@@ -165,11 +171,23 @@ const icon = computed(() => {
 | --- | --- | --- | --- |
 | color | 'white' \| 'black' \| 'light' \| 'dark' \| 'primary' \| 'link' \| 'success' \| 'danger' \| 'warning' \| 'info' | undefined | 色を指定します |
 | icon | string | undefined | iconのpathを指定します |
-| image | string | undefined | 画像のsrcに渡すpathを指定します |
+| image | string | undefined | 画像のsrcに渡すpathを指定します*1 |
 | rounded | 'none' \| 'small' \| 'medium' \| 'large' \| 'x-large' \| 'circle' | 'circle' | コンポーネントの`border-radius`を指定します |
 | size | 'x-small' \| 'small' \| 'medium' \| 'large' \| 'x-large' | "medium" | サイズを指定します |
 | variant | 'text' \| 'flat' \| 'elevated' \| 'tonal' \| 'outlined' \| 'plain' | 'flat' | コンポーネントに個別のスタイルを適用します |
 
+#### *1
+画像ファイルをimportしてモジュール名を指定することでコンポーネントに画像を適用します
```

Comment:
少し言葉足らずかなと感じました。以下でどうでしょう
```suggestion
バンドラにviteを利用している場合、以下の例のように、画像ファイルをimportしそのモジュール名を指定することで、viteによって解決されたpathを渡すことができます
```

Reply:
そうですね！そのように直します！🙆‍♀️